docs/developer-docs/smart-contracts/call/arguments.mdx

```diff
@@ -184,40 +194,14 @@ fn get_text(d: Day) -> &'static str {
 <AdornedTab value={"typescript"} label="TypeScript" endAdornment={<BetaChip />}>
 
 ```typescript
-import { Canister,  query, Variant } from 'azle';
-
-const Day = Variant({
-    Sun;
-    Mon;
-    Tue;
-    Wed;
-    Thu;
-    Fri;
-    Sat;
+export const ReactionType = IDL.Variant({
```

Comment:
Should this snippet be more full-featured like the others?

Reply:
Added full example